attributes/default.rb

```diff
@@ -21,9 +21,11 @@
 # The Datadog api key to associate your agent's data with your organization.
 # Can be found here:
 # https://app.datadoghq.com/account/settings
+# This can be a list of api keys
```

Comment:
You're right, it's not clear enough. I also changed list for array, as a list could be understood as a string `"apikey1, apikey2"`.